src/styles/main.scss

```
@@ -1,3 +1,4 @@
 @import "blocks/page";
 @import "blocks/header";
 @import "blocks/catalog";
+@import "./_variables"
```

To avoid errors variables should be first
```suggestion
@import "./_variables"
@import "blocks/page";
@import "blocks/header";
@import "blocks/catalog";
```